src/lib/pages/404/index.tsx

```diff
@@ -23,7 +23,7 @@ const Page404 = () => {
         </div>
 
         <div>
-          <Link href="/">Let&apos;s Go Back</Link>
+          <Link href="/" className="underline">Let&apos;s Go Back</Link>
```

Comment:
**suggestion**: how about on hover only? `hover:underline`

Reply:
@sozonome As per my understanding, the purpose of a template is to help user provide basic functionalities and boiler plate code setup in which they don't have to care about the basic user functionalities and accessibilities as well. Having said that, having a link underliner is not an opinionated styling. It is a general standard to which most of our human minds have been trained on. It increases the Usability and Accessibility of the website. I suggested the improvement solely because of that. Feel free to make any amends.

Reply:
The thought to hover only on the underline is also interesting, and useful in some ways. Although it won't be as intuitive as I want the 404 page to be. But yeah should be a useful add-on.

Reply:
Sure @surijatin I'm not completely disagree with your initial idea. I also believe template should have great defaults, but at the same time functionalities and a11y doesn't solely be the template responsibility. imho it will also be the dev / app writer responsibility onwards after scaffolding the project using the template as template is just a starting point.

Reply:
Kindly make sure husky is installed properly (run `pnpm i` then `pnpm lint:fix`)

Reply:
Form some reason I cannot push a branch from local to this repo. I had to make the edit directly on the file in this repo. I am not sure how to run the commands you have mentioned.

Reply:
Understood, let me help to guide you through.

As this repo is not owned by you, github automatically create a fork of this repo for you here: https://github.com/surijatin/nextarter-tailwind, specifically on this branch https://github.com/surijatin/nextarter-tailwind/tree/patch-1

You can clone the forked repo locally and make changes there

Reply:
That was very helpful, thanks a lot :)